.gitignore

```diff
@@ -46,3 +46,6 @@ venv.bak/
 
 # Test fixtures
 data/
+
+# vscode
+.vscode
```

Comment:
This should be in your global Git preferences, _e.g._ `$HOME/.gitignore`:

```gitignore
# Editors
.vscode
*.sw[p,o]
```

Reply:
Thanks